test/spec/modules/mint/mint1155.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { createMagicAdminSDK } from '../../../lib/factories';
+import { API_KEY } from '../../../lib/constants';
+import { createApiKeyMissingError, mintingError } from '../../../../src/core/sdk-exceptions';
+import { post } from '../../../../src/utils/rest';
+
+const successReq = Promise.resolve({
+    request_id: 'foo_123'
+});
+
+const failReq = Promise.resolve({
+    error: 'bar_456'
+});
+
+test('Successfully POSTs to 1155 minting endpoint', async () => {
+  const sdk = createMagicAdminSDK('https://example.com');
+
+  const postStub = jest.fn().mockImplementation(() => successReq);
+  (post as any) = postStub;
+
+  await expect(sdk.mint.startMint1155('0xfoo', 1, '0xbar', 0)).resolves.not.toThrow();
```

Comment:
Can we assert the result from this call?

Reply:
Good call, updated in latest